ait/core/server/handler.py

```diff
@@ -136,8 +141,12 @@ def handle(self, input_data):
         tlm_dict = tlm.getDefaultDict()
         packet_uid = tlm_dict[packet_name].uid
 
-        packet_data_length = int(binascii.hexlify(packet[10:12]), 16) + 1
+        packet_data_length = int(binascii.hexlify(packet[4:6]), 16) + 1
+        if (len(packet) < 6 + packet_data_length):
```

Comment:
@seanlu99 Is there any reason you can't change this to the following, by moving the +1 out of `packet_data_length`? I'm just curious why you split it up this way, as it is a little confusing to read the code with the check for `< 7` above and `< 6 + packet_data_length` here. 
```suggestion
        packet_data_length = int(binascii.hexlify(packet[4:6]), 16) 
        if (len(packet) < 7 + packet_data_length):
```

Reply:
I did it that way because the CCSDS documentation says that "A Space Packet shall consist of at least 7 octets" and the Packet Data Length is "one fewer than the length of the Packet Data Field." But now that you point it out I get why they subtracted 1 from the field. I'll change it to what you suggested.

Reply:
@aywaldron Wait actually if I do it the way you suggested then I have to do a `+2` on `udf_length` and `udf_start` still has a `+6` in it.